packages/catalog-generator/src/test/java/io/kaoto/camelcatalog/generators/EIPGeneratorTest.java

```diff
@@ -42,13 +43,12 @@ void setUp() throws IOException {
             camelYamlSchema = new String(is.readAllBytes(), StandardCharsets.UTF_8);
         }
 
-        eipGenerator = new EIPGenerator(camelCatalog, camelYamlSchema);
```

Comment:
Can we please restore this to be generated on each execution? it will be better to start from scratch on each test to ensure we're not modifying any schema from one test to another.